Create

BlueButterflies opened the pull request:
Greetings,
I followed the instructions as written in the instructions to be able to contribute to the translation.
I decided to contribute to the translation of the project into the Bulgarian language, so people who don't know any languages other than Bulgarian can read your valuable advice on collaborating on open source projects.

Thank you in advance

BlueButterflies commented:
Hi I don't understand why every time doing pull request it fails in the test part?

BlueButterflies commented:
Hi ,  I finished translating everything. I ask to be checked and approved. Thank you in advance.

BlueButterflies commented:
I don't understand, because I need git revert?

ipc103 commented:
> I don't understand, because I need to git revert and --no-gpg-sign?

Hi @BlueButterflies - you can ignore those comments as spam. Thank you for your contribution ❤️ Someone from our core team will review your contribution shortly and let you know if we have feedback. Thanks!

BlueButterflies commented:
> > I don't understand, because I need to git revert and --no-gpg-sign?
> 
> Hi @BlueButterflies - you can ignore those comments as spam. Thank you for your contribution ❤️ Someone from our core team will review your contribution shortly and let you know if we have feedback. Thanks!

Thank you

YahyaMohamedYahya reviewed (commented):
Thank you

jmeridth commented on .vs/opensource.guide/v17/DocumentLayout.json:
```diff
@@ -0,0 +1,12 @@
+{
+  "Version": 1,
+  "WorkspaceRootPath": "C:\\Users\\Dsvk2\\Source\\Repos\\opensource.guide\\",
```
same here re: specific to user.  Do we want? /cc @ahpook

jmeridth commented on .vs/VSWorkspaceState.json:
```diff
@@ -0,0 +1,7 @@
+  "ExpandedNodes": [
+    ""
+  ],
+  "SelectedNode": "\\C:\\Users\\Dsvk2\\Source\\Repos\\opensource.guide",
```
I imagine we're not going to want this line as it is specific to your user.  Correct @BlueButterflies?  Do we want this file committed? /cc @ahpook

ahpook replied:
nope, this shouldn't be added at all

BlueButterflies replied:
Thank you for report. I didn't see that I uploaded the .vs folder. I've already solved it, so you can proceed with the review

jmeridth commented on .vs/VSWorkspaceState.json:
```diff
@@ -0,0 +1,7 @@
+  ],
+  "SelectedNode": "\\C:\\Users\\Dsvk2\\Source\\Repos\\opensource.guide",
+  "PreviewInSolutionExplorer": false
+}
```
nit: if we keep, can we ensure newline at end of file please 🙏

BlueButterflies replied:
Can I remove this string from github???

BlueButterflies replied:
Thank you for report. I didn't see that I uploaded the .vs folder. I've already solved it, so you can proceed with the review.

jmeridth reviewed (commented):
@BlueButterflies thank you so much for the translation. :heart:  I have a few questions.

MarwanBz commented:
Hey @BlueButterflies have you figured out what caused the GitHub Actions CI / tests to failed?

BlueButterflies commented:
> Hey @BlueButterflies have you figured out what caused the GitHub Actions CI / tests to failed?

No .  I think what error came from that I asked for revision of the code. It had happened to me for the first time and therefore I asked

tuttheking81 commented:
> Greetings, I followed the instructions as written in the instructions to be able to contribute to the translation. I decided to contribute to the translation of the project into the Bulgarian language, so people who don't know any languages other than Bulgarian can read your valuable advice on collaborating on open source projects.
> 
> Thank you in advance

jmeridth commented on _articles/bg/starting-a-project.md:
```diff
@@ -0,0 +1,356 @@
+---
+lang: bg
+title: Стартиране на проект с отворен код
+description:Научете повече за света на отворения код и се пригответе да стартирате свой собствен проект.
```
need space after colon

jmeridth reviewed (commented):
~~I think this is what is causing the tests to fail.~~

BlueButterflies commented:
Hi, I corrected where there were spaces and changed the brackets that it told me in the test error. Only now it tells me again that there are errors and I can't figure out where to correct them. Thanks in advance

tomthorogood commented:
@BlueButterflies it looks like your links are incorrect. 
If you view your PR in "Rich diff" you can click on the links to see that they give a 404.

I believe your pattern needs to change from `../building-community/#dont-tolerate-bad-actors` should instead be (for example)`../building-community.md#dont-tolerate-bad-actors`

BlueButterflies commented:
> @BlueButterflies it looks like your links are incorrect. 
> If you view your PR in "Rich diff" you can click on the links to see that they give a 404.
> 
> I believe your pattern needs to change from `../building-community/#dont-tolerate-bad-actors` should instead be (for example)`../building-community.md#dont-tolerate-bad-actors`

Thank you, I'll try change

BlueButterflies commented:
> @BlueButterflies it looks like your links are incorrect. If you view your PR in "Rich diff" you can click on the links to see that they give a 404.
> 
> I believe your pattern needs to change from `../building-community/#dont-tolerate-bad-actors` should instead be (for example)`../building-community.md#dont-tolerate-bad-actors`

The problem persists
![Screenshot 2024-04-04 205742](https://github.com/github/opensource.guide/assets/52591976/7baefa0c-663d-4415-bb11-196da53f5cd4)

BlueButterflies commented:
Hi, I solved the problem with the links from the messages above in my "bg" folder, but the problem exists in a pcm/building-community folder as seen in the photo. The problem is located in the pcm/leadership-and-governance.md folder on line 76 shared ownership. 

Thank you in advance.

![Screenshot 2024-04-04 205742](https://github.com/github/opensource.guide/assets/52591976/e8a84eee-ef4a-4ce2-9d24-28349d83ad54)

jmeridth commented:
@BlueButterflies  That failing test is because you now have Bulgarian as the header](https://github.com/github/opensource.guide/pull/3209/files#diff-3739a510fe4cbd7ce59ebc70475318636bbb6287d6191ec08b99d7065d6a0c65R146) instead of English.  So [the link needs to have that as the text](https://github.com/github/opensource.guide/pull/3209/files#diff-864ce8b5dac5a64420313a6823a9018406c5eeb3ccbbaaba0aad05fcbc8a282eR172).

Hope this helps.

BlueButterflies commented:
> @BlueButterflies  That failing test is because [you now have Bulgarian as the header](https://github.com/github/opensource.guide/pull/3209/files#diff-3739a510fe4cbd7ce59ebc70475318636bbb6287d6191ec08b99d7065d6a0c65R146) instead of English.  So [the link needs to have that as the text](https://github.com/github/opensource.guide/pull/3209/files#diff-864ce8b5dac5a64420313a6823a9018406c5eeb3ccbbaaba0aad05fcbc8a282eR172).
> 
> Hope this helps.

I have already solved this problem in the "bg" folder I created.  Now the problem is in the pcm/leadership-and-governance.md line 76 folder which I did not create.

jmeridth commented:
@BlueButterflies @ahpook fixed that other translations with #3213 and we've merged main here. Now another issue in your translation. We're working to see about getting all issues at once but if you could fix that and see what comes next. Thank you for your patience and help. We're here to help also.

ahpook commented:
I'm working to fix the tests so they show the errors correctly - we'll need to bump the version of HTML-Proofer but there is a bug that needs to be fixed in that code first, or we will get lots of false positives from the non-ASCII languages. ( https://github.com/gjtorikian/html-proofer/issues/757 ) 

But there are still problems with the internal links on this translation, see below for the full list. 

Also it looks like at least the "building-community.md" doc was copied from the pcm/ directory. Was that the one you used as the basis for translation @BlueButterflies ? I think it would be hard to go directly from Nigerian Pidgin to Bulgarian and maybe some errors came in, might be worth a closer look.

For each of these errors you can either change the part after the `#` to the bulgarian version, _or_ find the header it targets and leave the header text in English.

```
For the Links > Internal check, the following failures were found:

* At _site/bg/best-practices/index.html:577:

  internally linking to ../building-community/#share-ownership-of-your-project; the file exists, but the hash 'share-ownership-of-your-project' does not

* At _site/bg/building-community/index.html:427:

  internally linking to ../starting-a-project/#writing-a-readme; the file exists, but the hash 'writing-a-readme' does not

* At _site/bg/building-community/index.html:428:

  internally linking to ../starting-a-project/#writing-your-contributing-guidelines; the file exists, but the hash 'writing-your-contributing-guidelines' does not

* At _site/bg/building-community/index.html:437:

  internally linking to ../how-to-contribute/#what-it-means-to-contribute; the file exists, but the hash 'what-it-means-to-contribute' does not

* At _site/bg/building-community/index.html:438:

  internally linking to ../best-practices/#sabi-to-say-no; the file exists, but the hash 'sabi-to-say-no' does not

* At _site/bg/building-community/index.html:529:

  internally linking to ../code-of-conduct/#how-you-go-take-think-your-code-of-conduct-to-stand-gidi-gba; the file exists, but the hash 'how-you-go-take-think-your-code-of-conduct-to-stand-gidi-gba' does not

* At _site/bg/building-community/index.html:610:

  internally linking to ../building-community/#no-dey-tolerate-bad-actors; the file exists, but the hash 'no-dey-tolerate-bad-actors' does not

* At _site/bg/building-community/index.html:626:

  internally linking to ../starting-a-project/#writing-a-readme; the file exists, but the hash 'writing-a-readme' does not

* At _site/bg/how-to-contribute/index.html:945:

  internally linking to #a-checklist-before-you-contribute; the file exists, but the hash 'a-checklist-before-you-contribute' does not

* At _site/bg/leadership-and-governance/index.html:442:

  internally linking to ../how-to-contribute/#what-it-means-to-contribute; the file exists, but the hash 'what-it-means-to-contribute' does not

* At _site/bg/leadership-and-governance/index.html:475:

  internally linking to ../building-community/#share-ownership-of-your-project; the file exists, but the hash 'share-ownership-of-your-project' does not

* At _site/bg/legal/index.html:472:

  internally linking to #does-my-project-need-an-additional-contributor-agreement; the file exists, but the hash 'does-my-project-need-an-additional-contributor-agreement' does not

* At _site/bg/legal/index.html:472:

  internally linking to #what-does-my-companys-legal-team-need-to-know; the file exists, but the hash 'what-does-my-companys-legal-team-need-to-know' does not

* At _site/bg/legal/index.html:488:

  internally linking to #which-open-source-license-is-appropriate-for-my-project; the file exists, but the hash 'which-open-source-license-is-appropriate-for-my-project' does not

* At _site/bg/legal/index.html:528:

  internally linking to #which-open-source-license-is-appropriate-for-my-project; the file exists, but the hash 'which-open-source-license-is-appropriate-for-my-project' does not

* At _site/bg/legal/index.html:534:

  internally linking to #which-open-source-license-is-appropriate-for-my-project; the file exists, but the hash 'which-open-source-license-is-appropriate-for-my-project' does not

* At _site/bg/legal/index.html:537:

  internally linking to ../starting-a-project/#avoiding-name-conflicts; the file exists, but the hash 'avoiding-name-conflicts' does not

* At _site/bg/legal/index.html:574:

  internally linking to ../leadership-and-governance/#do-i-need-a-legal-entity-to-support-my-project; the file exists, but the hash 'do-i-need-a-legal-entity-to-support-my-project' does not

* At _site/bg/starting-a-project/index.html:420:

  internally linking to #does-open-source-mean-free-of-charge; the file exists, but the hash 'does-open-source-mean-free-of-charge' does not

BlueButterflies commented:
Thanks guys for your replies. I'll take a look at the entire "bg" folder again.

alvian009 reviewed (commented):
isps,[]( 10:1 or 5:1 or 1:1 
[Rar.txt](https://github.com/github/opensource.guide/files/14967465/Rar.txt)
)

Aleis007891 reviewed (commented):
Hol

Sasa12sasa reviewed (commented):
[redacted as spam]

Sasa12sasa commented:
جيد جدا

mbdahhan commented:
King import export

mbdahhan commented:
![Uploading IMG_0531.jpeg…]()

BlueButterflies commented:
Hi guys, I finally managed to resolve the link connections, thanks also to your interventions.
You were helpful, especially because I didn't understand what was wrong with the links.
Thanks again.
I'm happy to have been part of the translation of the site.

jmeridth commented:
@BlueButterflies this looks good to me but want @ahpook to give final 👍 .  Thank you for all your work and patience on this.

BlueButterflies commented:
> @BlueButterflies this looks good to me but want @ahpook to give final 👍 .  Thank you for all your work and patience on this.

Thank you

2toa commented:
What time do you want me to come to

BlueButterflies commented:
@ahpook please accepts pull requests

BlueButterflies commented:
> @BlueButterflies this looks good to me but want @ahpook to give final 👍 .  Thank you for all your work and patience on this.

Hi, @jmeridth Sorry for the inconvenience, I'm wondering what to do with the pull requests in the open.guide repository in my account, since @ahpook linked everything to me. Because then I should make pull requests to the open-giude repository owned by github. 
I hope I explained myself well. Thank you in advance.

Gopstop92 reviewed (commented):
## Transcript

```
Welcome to Termux!

Docs:       https://termux.dev/docs
Donate:     https://termux.dev/donate
Community:  https://termux.dev/community

Working with packages:

 - Search:  pkg search <query>
 - Install: pkg install <package>
 - Upgrade: pkg upgrade

Subscribing to additional repositories:

 - Root:    pkg install root-repo
 - X11:     pkg install x11-repo

For fixing any repository issues,
try 'termux-change-repo' command.

Report issues at https://termux.dev/issues
~ $
```
##


## Termux App Info

**APP_NAME**: `Termux`  
**PACKAGE_NAME**: `com.termux`  
**VERSION_NAME**: `0.118.0`  
**VERSION_CODE**: `118`  
**TARGET_SDK**: `28`  
**IS_DEBUGGABLE_BUILD**: `false`  
**APK_RELEASE**: `F-Droid`  
**SIGNING_CERTIFICATE_SHA256_DIGEST**: `228FB2CFE90831C1499EC3CCAF61E96E8E1CE70766B9474672CE427334D41C42`  
##


## Device Info

### Software

**OS_VERSION**: `4.19.127-perf-g8a5c8b83a8c1`  
**SDK_INT**: `30`  
**RELEASE**: `11`  
**ID**: `RP1A.200720.011`  
**DISPLAY**: `RP1A.200720.011`  
**INCREMENTAL**: `V12.5.12.0.RCZRUXM`  
**SECURITY_PATCH**: `2024-02-01`  
**IS_DEBUGGABLE**: `0`  
**IS_TREBLE_ENABLED**: `true`  
**TYPE**: `user`  
**TAGS**: `release-keys`  

### Hardware

**MANUFACTURER**: `Xiaomi`  
**BRAND**: `Redmi`  
**MODEL**: `220233L2G`  
**PRODUCT**: `dandelion_ru2`  
**BOARD**: `dandelion`  
**HARDWARE**: `mt6762`  
**DEVICE**: `dandelion`  
**SUPPORTED_ABIS**: `armeabi-v7a, armeabi`  
##


## Termux APT Info


### Subscribed Repositories

#### sources.list
`deb https://mirrors.zju.edu.cn/termux/apt/termux-main stable main`
#### root-repo (sources.list.d/root.list)
`deb https://mirrors.zju.edu.cn/termux/apt/termux-root root stable`  
#### x11-repo (sources.list.d/x11.list)
`deb https://mirrors.zju.edu.cn/termux/apt/termux-x11 x11 main`  
##


### Updatable Packages

```
dbus/stable 1.15.6-2 arm [upgradable from: 1.15.6-1]
ed/stable 1.20.2 arm [upgradable from: 1.20.1]
fribidi/stable 1.0.14 arm [upgradable from: 1.0.13]
libtsduck/stable 3.37.3670 arm [upgradable from: 3.36.3528]
mesa/stable 24.0.6 arm [upgradable from: 24.0.5]
pango/stable 1.52.2 arm [upgradable from: 1.50.14]
tsduck-tools/stable 3.37.3670 arm [upgradable from: 3.36.3528]
```
##



##


## Termux Files Info


**TERMUX_REQUIRED_FILES_DIR_PATH ($PREFIX)**: `/data/data/com.termux/files`  
**ANDROID_ASSIGNED_FILES_DIR_PATH**: `/data/user/0/com.termux/files`  

```
$ echo 'ls info:'
/system/bin/ls -lhdZ '/data/data' '/data/user/0' '/data/data/com.termux' '/data/user/0/com.termux' '/data/data/com.termux/files' '/data/user/0/com.termux/files' '/data/user/0/com.termux/files' '/data/user/com.termux/files' '/data/data/com.termux/files/usr-staging' '/data/data/com.termux/files/usr' '/data/data/com.termux/files/home' '/data/data/com.termux/files/usr/bin/login' 2>&1
echo; echo 'mount info:'
/system/bin/grep -E '( /data )|( /data/data )|( /data/user/[0-9]+ )' /proc/self/mountinfo 2>&1 | /system/bin/grep -v '/data_mirror' 2>&1

ls info:
ls: /data/user/com.termux/files: No such file or directory
ls: /data/data/com.termux/files/usr-staging: No such file or directory
drwxrwx--x 322 system  system  u:object_r:system_data_file:s0                   52K 2024-04-27 00:58 /data/data
drwx------   6 u0_a301 u0_a301 u:object_r:app_data_file:s0:c45,c257,c512,c768  3.4K 2024-04-26 00:47 /data/data/com.termux
drwxrwx--x   4 u0_a301 u0_a301 u:object_r:app_data_file:s0:c45,c257,c512,c768  3.4K 2024-04-12 23:45 /data/data/com.termux/files
drwx------   4 u0_a301 u0_a301 u:object_r:app_data_file:s0:c45,c257,c512,c768  3.4K 2024-04-23 09:54 /data/data/com.termux/files/home
drwx------  11 u0_a301 u0_a301 
![Screenshot_2024-04-23-08-57-33-134_com termux](https://github.com/github/opensource.guide/assets/108019142/7fe3738a-8aaa-4c71-9b29-d2bb2b9ba72d)

jmeridth commented:
@BlueButterflies there may be a bit more work needed. I'll confirm with @ahpook. Thank you again for your work and patience.

ahpook commented:
Yep this LGTM - I reran tests locally with the updated HTML-Proofer and all the internal links work. Great work @BlueButterflies , thank you for responding to all the issues.

ahpook reviewed (approved):
Clean checks w/html-proofer 5.0.8

BlueButterflies commented:
> Clean checks w/html-proofer 5.0.8

Thank you for accepting my contribution 
Sorry I ask again. in the open.guide repository in my account I have two pull requests. I merge and send to origin main with revision request and then you accept? 


Thank you in advance

ahpook commented:
> Sorry I ask again. in the open.guide repository in my account I have two pull requests. I merge and send to origin main with revision request and then you accept?

Sorry! I did not understand the question before.

I looked at these PRs : https://github.com/BlueButterflies/opensource.guide/pulls 

They are both spam from fake accounts :( so you can just close them.

BlueButterflies commented:
> > Sorry I ask again. in the open.guide repository in my account I have two pull requests. I merge and send to origin main with revision request and then you accept?
> 
> Sorry! I did not understand the question before.
> 
> I looked at these PRs : https://github.com/BlueButterflies/opensource.guide/pulls 
> 
> They are both spam from fake accounts :( so you can just close them.

 Thank you

BlueButterflies commented:
> Oh damn, there is one more: 
> 
> ```
> 
> * At _site/bg/how-to-contribute/index.html:945:
> 
>   internally linking to #a-checklist-before-you-contribute; the file exists, but the hash 'a-checklist-before-you-contribute' does not exist
> ```
> 
> This is at line 500 in how-to-contribute.md

JhonyM20 commented:
> Greetings,
> I followed the instructions as written in the instructions to be able to contribute to the translation.
> I decided to contribute to the translation of the project into the Bulgarian language, so people who don't know any languages other than Bulgarian can read your valuable advice on collaborating on open source projects.
> 
> Thank you in advance
>

BlueButterflies reviewed (commented):
Correction of link and some untranslated sentences in the how-to-contribute file found by @ahpook

ahpook commented:
I had to fix one small thing in the data structure in #3215 but this is now live on the site! 

https://opensource.guide/bg/  

Thank you @BlueButterflies for all your work on this 💯

BlueButterflies commented:
> I had to fix one small thing in the data structure in #3215 but this is now live on the site! 
> 
> https://opensource.guide/bg/  
> 
> Thank you @BlueButterflies for all your work on this 💯 
> 
>
